### PR TITLE
[General] Create lvim.desktop

### PR DESCRIPTION
**Pull request by UltimateOmega**
# Description
This PR will add desktop file for lunarvim

### CODE REVIEW

**Comment by UltimateOmega** [comment]
@ChristianChiarulli We need LunarVim icon for desktop file.

File: utils/installer/uninstall.sh [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 rm -rf .local/share/lunarvim
 sudo rm /usr/local/bin/lvim
+rm -rf .local/share/applications/lvim.desktop
```
Comment (abzcoding) [inline utils/installer/uninstall.sh:5]: while at it, can you make these `~/.local` instead of `.local`

Comment (UltimateOmega) [reply utils/installer/uninstall.sh]: done!

**Review by abzcoding — APPROVED** [review]

**Comment by abzcoding** [comment]
one more thing, we don't need this in mac os, right?
should we make an exception for mac or not 🤔

**Comment by UltimateOmega** [comment]
> one more thing, we don't need this in mac os, right?
> should we make an exception for mac or not 🤔

I don't use mac os, so I have no idea.

**Review by abzcoding — APPROVED** [review]
LGTM 🚀

**Comment by ChristianChiarulli** [comment]
Hopefully Simon will have an icon soon

**Pull request merged**